Adyen/Helpers/UIViewConstraintsHelper.swift

```diff
@@ -12,48 +12,79 @@ import UIKit
-    /// - IMPORTANT: Both views should be in the same hierarchy.
+    /// - IMPORTANT: The view & layout guide must be in the same hierarchy.
     /// - Parameter margins: The layout guide to constraint to.
     /// - Parameter padding: Padding values for each edge. Default is 0 on all edges.
     @discardableResult
-    public func anchor(inside margins: UILayoutGuide, with padding: UIEdgeInsets = .zero) -> [NSLayoutConstraint] {
-        anchor(inside: .layoutGuide(margins), edgeInsets: EdgeInsets(edgeInsets: padding))
+    public func anchor(
+        inside margins: UILayoutGuide,
+        with padding: UIEdgeInsets = .zero
+    ) -> [NSLayoutConstraint] {
+        anchor(
+            inside: .layoutGuide(margins),
+            edgeInsets: EdgeInsets(edgeInsets: padding)
+        )
     }
     
     /// Attaches the given edges of this view to corresponding anchors inside the specified anchor source.
+    /// - IMPORTANT: The view & anchor source must be in the same hierarchy.
     /// - Parameters:
     ///   - anchorSource: The anchor source to contain this view.
     ///   - edgeInsets: Edges with inset values on which the views should be anchored. Defaults to all 4 edges with 0 inset each.
     @discardableResult
-    public func anchor(inside anchorSource: LayoutAnchorSource,
-                       edgeInsets: EdgeInsets = .zero) -> [NSLayoutConstraint] {
+    public func anchor(
+        inside anchorSource: LayoutAnchorSource,
+        edgeInsets: EdgeInsets = .zero
+    ) -> [NSLayoutConstraint] {
         base.translatesAutoresizingMaskIntoConstraints = false
         
         var constraints: [NSLayoutConstraint] = []
         if let top = edgeInsets.top {
-            constraints.append(base.topAnchor.constraint(equalTo: anchorSource.anchorSet.topAnchor,
-                                                         constant: top))
+            constraints.append(
+                base.topAnchor.constraint(
+                    equalTo: anchorSource.anchorSet.topAnchor,
+                    constant: top
+                )
+            )
         }
         if let left = edgeInsets.left {
-            constraints.append(base.leadingAnchor.constraint(equalTo: anchorSource.anchorSet.leadingAnchor,
-                                                             constant: left))
+            constraints.append(
+                base.leadingAnchor.constraint(
+                    equalTo: anchorSource.anchorSet.leadingAnchor,
+                    constant: left
+                )
+            )
         }
         if let bottom = edgeInsets.bottom {
-            constraints.append(base.bottomAnchor.constraint(equalTo: anchorSource.anchorSet.bottomAnchor,
-                                                            constant: bottom))
+            constraints.append(
+                base.bottomAnchor.constraint(
+                    equalTo: anchorSource.anchorSet.bottomAnchor,
+                    constant: -bottom
```

Comment:
Not sure if I agree with changing the calling of this function from positive to negative, but if we agree to go with it then maybe we can reverse the anchors for bottom/trailing so we don't provide a negative anywhere like:
```
constraints.append(
                anchorSource.anchorSet.bottomAnchor.constraint(
                    equalTo: base.bottomAnchor, constant: bottom
                )
            )
```